reana/reana_dev/run.py

```diff
@@ -538,28 +555,46 @@ def run_example(
                             or "stopped" in status
                         ):
                             break
-                    # verify logs message presence
-                    for log_message in get_expected_log_messages_for_example(component):
-                        cmd = "reana-client logs -w {0} | grep -c '{1}'".format(
-                            workflow_name, log_message
+
+                # check only is here
+                if not submit_only:
+                    cmd = f"reana-client status -w {workflow_name}"
+                    status = run_command(cmd, component, return_output=True)
+
+                    if "running" in status:
+                        run_statistics["running"].append(workflow_name)
+
+                    if "failed" in status:
+                        run_statistics["failed"].append(workflow_name)
+                        continue
+
+                    if "finished" in status or "stopped" in status:
+
+                        if not _verify_log_output(component, workflow_name):
+                            run_statistics["failed"].append(workflow_name)
+                            continue
+
+                        missing_files = _return_missing_output_files(
+                            component, workflow_name
                         )
-                        run_command(cmd, component)
-                    # verify output file presence
-                    cmd = "reana-client ls -w {0}".format(workflow_name)
-                    listing = run_command(cmd, component, return_output=True)
-                    click.secho(listing)
-                    expected_files = file or get_expected_output_filenames_for_example(
-                        component
-                    )
-                    for expected_file in expected_files:
-                        if expected_file not in listing:
-                            click.secho(
-                                "[ERROR] Expected output file {0} not found. "
-                                "Exiting.".format(expected_file)
-                            )
-                            sys.exit(1)
-    # report that everything was OK
-    run_command("echo OK", component)
+
+                        if len(missing_files):
+                            run_statistics["failed"].append(workflow_name)
+                        else:
+                            run_statistics["passed"].append(workflow_name)
+
+    if not submit_only:
+        failed_report = ""
+        if len(run_statistics["failed"]):
+            failed_report = f"({','.join(run_statistics['failed'])})"
+
+        report_message = (
+            f"\nreana-dev run-example demo summary:\n"
+            f"  - {len(run_statistics['running'])} workflows are running\n"
+            f"  - {len(run_statistics['passed'])} passed\n"
+            f"  - {len(run_statistics['failed'])} failed {failed_report}\n"
```

Comment:
Cosmetics: When some jobs fail, they are printed as:

```
(helloworld-yadage-kubernetes,helloworld-snakemake-kubernetes,helloworld-cwl-kubernetes,helloworld-serial-kubernetes)
```

We could separate them with spaces, so that they would be easier to read and copy-paste via double-clicking-to-select-word.  So something like `(helloworld-yadage-kubernetes, helloworld-snakemake-kubernetes, helloworld-cwl-kubernetes, helloworld-serial-kubernetes)`.